03-task-manager/starter/controllers/tasks.js

```diff
@@ -0,0 +1,84 @@
+const Task = require("../models/Task");
+
+//we want to wrap all of our async functions using asyncWrapper in this file
+const asyncWrapper = require("../middleware/async");
+const { createCustomError } = require("../errors/custom-error");
+
+//we completely removed the try/catch block and put it in the async.js file
+const getAllTasks = asyncWrapper(async (req, res) => {
+  const tasks = await Task.find({}); //find is a static function
+  //status options below
+  //res.status(200).json({ tasks });
```

Comment:
Looks good, just remember to remove comments before submitting your HW assignments.  Remember you can always 'stash' if you want to save some comments or changes locally!